bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,222 @@
+# MagentaTV Binding (aka TelekomTV aka EntertainTV)
+
+---
+
+## Release: 2.5-pre5
+
+This Binding allows controlling the Deutsche Telekom Magenta TV receiver series Media Receiver MR4xx and MR2xx (Telekom NGTV / Huawei Envision platform). 
+The Binding does NOT support MR3xx/1xx (old Entertain system based on Microsoft technology)!
+
+The binding has been tested with the EntertainTV service as well as the new MagentaTV service (launched in 10/2018).
+
+This include device discovery, sending keys from the remote and also receiving program events.
+
+This binding replaces the former versions EntertainTV and TelekomTV bindings.
+
+---
+
+Please check out the [openHAB community thread](https://community.openhab.org/t/magentatv-entertaintv-binding-for-deutsche-telekom-mr-3xx-and-4xx) and discuss your ideas, requests and technical problems with the community.
+
+---
+
+### Supported Models
+
+* Deutsche Telekom Media Receiver MR401B - fully supported
+* Deutsche Telekom Media Receiver MR201  - fully supported
+* Deutsche Telekom Media Receiver MR400  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR200  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR3xx  - NOT supported (different platform)
+* Deutsche Telekom Media Receiver MR1xx  - NOT supported (different platform)
+
+
+### Receiver Standby Mode
+
+The Media receiver has 3 different standby modes, which could be selected in the receiver's settings.
+
+- Standby - full standby, receiver active all the time
+- Suspend/Resume - The receiver goes to sleep mode, but could be awaked by a Wake-on-LAN packet
+- Shutdown - Powering off will shutdown the receiver, can be awaked only with the power button
+
+Standby provides the best results, because the binding could wakeup the receiver (Power On/Off). Suspend/Resume would require a Wake-on-LAN, which could be done, but is currently not implemented. Shutdown turns the receiver off, which requires a manual PowerOn.
```

Comment:
Please add a line break after every sentence throughout this README.

Reply:
done